src/blant.c

```diff
@@ -1379,14 +1381,15 @@ int main(int argc, char *argv[])
 	    if(_confidence >= 1) _confidence /= 100; // user specified percent
 	    break;
 	case 'k': _k = atoi(optarg);
+		_k_base = MIN(_k, 8);
```

Comment:
OK I think I see what k_base is now. Still, add a detailed comment at the location where it's declared.